docs/guides/integration.md

```diff
@@ -96,61 +93,71 @@ Ouput:
 [info] Run completed in 2 seconds, 954 milliseconds.
 [info] Total number of tests run: 1
 ```
+{% sample lang="Service Dependencies" %}
+
+The Docker compose specifies the application's dependencies.  It is useful for local development. It can be started and stopped using `docker-compose up -d` and `docker-compose down`.
+This will also be essential for our Earthly integration tests.
+
+Docker Compose:
+``` yaml
+version: "3"
+services:
+  postgres:
+    container_name: local-postgres
+    image: aa8y/postgres-dataset:iso3166
+    ports:
+      - 5432:5432
+    hostname: postgres
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  postgres-ui:
+    container_name: local-postgres-ui
+    image: adminer:latest
+    depends_on:
+      - postgres
+    ports:
+      - 8080:8080
+    hostname: postgres-ui
+```
+
+
 {% endmethod %}
 
-## The Earthfile
+### The Basic Earthfile
 
-We start with a simple Earthfile that can build and create a docker image for our app. See [Basic](./basics) guide for more details on that.
+We start with a simple Earthfile that can build and create a docker image for our app. See the [Basic](./basics) guide for more details, as well as examples in many programming languages.
```

Comment:
Add link to examples section here?

Reply:
There is a link to the Basics guide. Those were the examples I was referring to, in this case.

Reply:
Ah gotcha